sim/battle-actions.ts

```diff
@@ -1902,6 +1902,13 @@ export class BattleActions {
 			this.battle.singleEvent('End', this.dex.abilities.get('Illusion'), pokemon.abilityState, pokemon);
 		}
 
+		if (pokemon.species.name === 'Morpeko-Hangry') {
+			pokemon.baseSpecies = this.battle.dex.species.get('Morpeko-Hangry');
+			pokemon.details = pokemon.species.name + (pokemon.level === 100 ? '' : ', L' + pokemon.level) +
+				(pokemon.gender === '' ? '' : ', ' + pokemon.gender) + (pokemon.set.shiny ? ', shiny' : '');
+			this.battle.add('detailschange', pokemon, pokemon.details);
```

Comment:
Does this not generate extraneous text and animations? (Although I guess we don't currently have any other way of telling the client that what we though was a temporary forme change turned out to be permanent. Or maybe there's a way to silence this.)

Reply:
i can probably make detailschange support  '[silent]' if needed

Reply:
do that !!!